app/api/resources/mentorship_relation.py

```diff
@@ -100,8 +102,14 @@ def put(cls, request_id):
         user_id = get_jwt_identity()
         response = DAO.accept_request(user_id=user_id, request_id=request_id)
 
-        return response
+        if response[1] is 200:
+            request = MentorshipRelationModel.find_by_id(request_id)
+            sender_id = request.action_user_id
+            sender = UserModel.find_by_id(sender_id)
+            email = sender.email
```

Comment:
You can directly pass sender.email in arguments of email function, makes the code look much more clean and simple.